pytest_redis/factories/proc.py

```diff
@@ -92,6 +94,13 @@ def redis_proc_fixture(
         else:
             redis_datadir = tmp_path_factory.mktemp(f"pytest-redis-{request.fixturename}")
 
+        if modules:
+            redis_modules = modules
+        elif config.get("modules"):
+            redis_modules = config["modules"].split(",")
```

Comment:
@mguijarr the problem in this line is that having the `elif config.get("modules"):` in the line above does not tell mypy, that `config["modules"]` holds any data. Could be something to address within mypy, though looking at the config dict, the "modules" is always set, changing the condition in line 99 to `elif config["modules"]:` makes mypy understand, that if it gets into line 100, the value under modules is always string, and is not possible to be None.

Reply:
Ok! I see. Thanks a lot for fixing this!